root/release/edit/tracklist.tt

```diff
@@ -328,6 +325,20 @@
        %]
     </table>
 
+    <!-- ko if: hasTooEarlyFormat -->
+      <div class="warning">
+        [%- warning_icon %]
+        <p>
+          <strong>[% l('Warning:') | html_entity %]</strong>
+          [% l('This medium is set to a format that did not exist on the selected release date.') %]
+          [% l('This should never be correct. Make sure that, say, your release set to CD isn’t meant to be a vinyl, or your vinyl isn’t actually a shellac disc.') %]
```

Comment:
The release has not been entered by the current editor, so we should probably not write “your” and it should be clarified that the mistake is not the fault of the current editor either:

```suggestion
          [% l('This mistake was there before you started editing this release. Make sure that, say, the release set to CD isn’t meant to be a vinyl, or the vinyl isn’t actually a shellac disc.') %]
```

Reply:
We don't usually clarify this elsewhere (and then we would need to change the warning text depending on whether the editor added it now or not). Is it really needed? It's still true that it should never be correct. 

The 'the' is certainly better :)

Reply:
> We don't usually clarify this elsewhere (and then we would need to change the warning text depending on whether the editor added it now or not).

Changing release format is not straightforward because releases can be mixed together and then changing only one field in only one release is not solving anything, it often requires to check the release group instead. So I feel that more explanations should be given about it. Maybe it should even be discouraged to fix it if information is missing, e.g. “Please fix it if and only if you have enough information to distinguish between possible mixed information from different releases.”

> It's still true that it should never be correct.

Correct. I suggested to start with “This mistake” instead which is probably clear enough (as a mistake should never be correct)?

Reply:
I added a separate wording for when the error predates the editing session. See the PR description for images too.